src/Util/ObjectAclManipulator.php

```diff
@@ -50,7 +81,7 @@ public function batchConfigureAcls(OutputInterface $output, AdminInterface $admi
             $objectIds = [];
             $objectIdIterator = new \ArrayIterator();
 
-            foreach ($qb->getQuery()->iterate() as $row) {
+            foreach ($qb->getQuery()->getIterator() as $row) {
```

Comment:
`iterate` is deprecated: https://github.com/doctrine/mongodb-odm/blob/1.3.0/lib/Doctrine/ODM/MongoDB/Query/Query.php#L413